apps/crc_interactive.py

```diff
@@ -77,6 +77,13 @@ def __init__(self) -> None:
         additional_args.add_argument('-f', '--feature', help='specify a feature, e.g. `ti` for GPUs')
         additional_args.add_argument('-o', '--openmp', action='store_true', help='run using OpenMP style submission')
 
+    def parse_time(time_str: str) -> time:
```

Comment:
My original suggestion was to define this as a top level function, but your implementation as method is equally valid. The only difference is that this method should be made static if included as a method:

```python
@staticmethod
def parse_time(time_str: str) -> time:
    ...
```

If left as is, the parent class will pass `self` as the first argument, which is not included in the signature (nor should it be in this case).

Leave this class as a method, but add the static tag.